docs/source/about/guides/app_examples.rst

```diff
@@ -200,11 +200,11 @@ Changing Resolution and Refresh Rate
 
    .. tab:: X11
 
-      +----------------------+------------------------------------------------------------------------------------------------------------------------------------+
-      | Command Preparations | Do: ``sh -c "xrandr --output HDMI-1 --mode \"${SUNSHINE_CLIENT_WIDTH}x${SUNSHINE_CLIENT_HEIGHT}\" --rate ${SUNSHINE_CLIENT_FPS}"`` |
```

Comment:
I removed the `\"` construct. It works fine without. But I can't test the same simplification for wlr-xrandr so I did not modify that one.